src/main/resources/assets/curios/lang/pl_pl.json

```diff
@@ -11,6 +11,10 @@
   "commands.curios.clearAll.success": "Wyczyszczono wszystkie sloty graczowi %s",
   "commands.curios.reset.success": "Zresetowano sloty graczowi %s",
   "argument.curios.type.unknown": "Nieznany typ slotu: %s",
+  "item.curios.amulet": "Amulet Curious",
+  "item.curios.ring": "Pierścień Curious",
+  "item.curios.crown": "Korona Curious",
+  "item.curios.knuckles": "Curious Knuckles",
```

Comment:
I have a problem with translating this. Could you please send me a link to a real life photo of this thing? It should help a lot, as the information I have now besides the name (i.e. the low-res Minecraft icon and the fact that it is something you wear on your hands) is unfortunately not enough to make sure I translated it correctly.

Reply:
It's something like [this](https://imgr.search.brave.com/wQgHbe13kXiU-5ZdPwx0UTtfirls_ZMoOFr_OW0_9Kc/fit/1200/1200/ce/1/aHR0cDovL3d3dy50/aGVzcGVjaWFsaXN0/c2x0ZC5jb20vZmls/ZXMvQnJhc3NfS251/Y2tsZXMuanBn).

The name is a pun, which admittedly isn't easy to translate. It's because Curios sounds like curious. It's important that it's curious and not Curios because curious is an adjective describing the item. In this context, "curious" is supposed to mean "interesting" or "mysterious".

Reply:
Thank you for the explanation, it clarifies a lot.
It seems that I wrongly assumed that since those 4 items are just debug items, you won't come up with a fancy name for them, but instead use something simple like this schema: `<your_mod_name> + <item_name>`.